assets/newspack-ui/style.scss

```diff
@@ -0,0 +1,35 @@
+@use 'scss/variables';
+@use 'scss/elements';
+@use 'scss/modals';
+
+.newspack-ui {
+	font-family: var( --newspack-ui-font-family );
+	font-size: var( --newspack-ui-font-size-medium ) !important;
+
+	// Need better spot for this
+	&__color-text-gray {
+		color: var( --newspack-ui-color-text-med-contrast );
+	}
+
+	// Overrides for theme styles -- to move to theme?
+	h2,
+	h3 {
+		font-size: var( --newspack-ui-font-size-medium );
+	}
+
+	a,
+	a:hover {
+		color: inherit !important;
+	}
+
+	// TODO: This specifically is a WooCommerce HTML structure; have been using it for other elements so far, but do we want to?
```

Comment:
We might realize at some point that the newspack-ui should have a `woocommerce/` directory with all the Woo-specific required tweaks.

Reply:
That's a really good point, and is kind of on trend for what we've had to do for things like the theme. I spun up a seperate /woocommerce directory, and added these styles to an `_overrides.scss` file inside of there, since they're just undoing styles WC is adding so far.

Reply:
Updated in [d81a167](https://github.com/Automattic/newspack-plugin/pull/2810/commits/d81a16709a64af9cae5fb89564ebdbc5acfd8c8b)!